packages/manager/modules/pci/src/components/pci-order-command/pci-order-command.html

```diff
@@ -2,10 +2,13 @@ <h5 data-translate="pci_order_command_title"></h5>
 <div>
     <ng-transclude></ng-transclude>
 </div>
-<tail-logs data-ng-class="{'overloader_height_tail_logs': $ctrl.autoHeight }">
-    <p data-ng-bind="$ctrl.orderApiUrl"></p>
+<div
+    class="command oui-color-p-000 p-2"
+    data-ng-class="{'overloader_height_tail_logs': $ctrl.autoHeight }"
+>
+    <p data-ng-bind="$ctrl.orderApiUrl" class="api-url monospace"></p>
```

Comment:
Don't hesitate/forget to have one timebinding when values are immutable.
 ```suggestion
    <p data-ng-bind=":: $ctrl.orderApiUrl" class="api-url monospace"></p>
```

Reply:
I think that we want the `pci-order-component` to be able to handle dynamic values here, for exemple it can be used to describe different endpoints configuration.